client/www/pages/api/uptime.ts

```diff
@@ -0,0 +1,49 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const UPTIMEROBOT_API_KEY = process.env.UPTIMEROBOT_API_KEY!;
+const UPTIMEROBOT_API_URL = 'https://api.uptimerobot.com/v2/getMonitors';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  if (req.method !== 'GET') {
+    return res.status(405).json({ error: 'Method not allowed' });
+  }
+
+  try {
+    const response = await fetch(UPTIMEROBOT_API_URL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        api_key: UPTIMEROBOT_API_KEY,
+        format: 'json',
+        logs: 1,
+        logs_limit: 200,
+        response_times: 1,
+        response_times_limit: 90,
+        custom_uptime_ratios: '1-7-30-90',
+        custom_uptime_ranges: Array.from({ length: 90 }, (_, i) => {
+          const start = Math.floor(Date.now() / 1000) - (i + 1) * 24 * 60 * 60;
```

Comment:
What is `custom_uptime_ranges` supposed to calculate? 

Would be good if we could make this more descriptive, as it's a bit hard to understand.